tests/backbeat/tests/api/metrics.js

```diff
@@ -27,49 +31,90 @@ function makeDelayedGETRequest(endpoint, cb) {
     }, 5000);
 }
 
-describe.skip('Backbeat replication metrics', function dF() {
+describe('Backbeat replication metrics', function dF() {
     this.timeout(REPLICATION_TIMEOUT);
     const roleArn = 'arn:aws:iam::root:role/s3-replication-role';
+    const storageClass = `${destAWSLocation},${destAzureLocation}`;
+    const pathPrefix = '/_/backbeat/api/metrics/crr';
 
     beforeEach(done => series([
         next => scalityUtils.createVersionedBucket(srcBucket, next),
         next => scalityUtils.putBucketReplicationMultipleBackend(srcBucket,
-            destBucket, roleArn, destLocation, next),
+            'placeholder', roleArn, storageClass, next),
     ], done));
 
-    afterEach(done => series([
+    afterEach(done => parallel([
         next => scalityUtils.deleteVersionedBucket(srcBucket, next),
-        next => awsUtils.deleteAllVersions(destBucket,
-            `${srcBucket}/${keyPrefix}`, next),
+        next => awsUtils.deleteAllVersions(awsDestBucket, destKeyPrefix, next),
+        next => scalityUtils.deleteAllBlobs(destContainer, destKeyPrefix, next),
     ], done));
 
-    it('should return specific object properties', done => {
-        makeGETRequest('/_/backbeat/api/metrics/crr/all', (err, res) => {
-            assert.ifError(err);
-            assert.equal(res.statusCode, 200);
-            getResponseBody(res, (err, body) => {
+    [
+        `${pathPrefix}/all`,
+        `${pathPrefix}/${destAWSLocation}`,
+        `${pathPrefix}/${destAzureLocation}`,
+    ].forEach(path => {
+        it(`should return all metrics for path ${path}`, done => {
+            makeGETRequest(path, (err, res) => {
                 assert.ifError(err);
-                const metricTypes = ['backlog', 'completions',
-                    'throughput', 'failures'];
-                const keys = Object.keys(body);
-                keys.forEach(key => {
-                    assert(metricTypes.includes(key));
-                    assert(body[key].description);
-                    assert(body[key].results);
-                    const resultKeys = Object.keys(body[key].results);
+                assert.equal(res.statusCode, 200);
+                getResponseBody(res, (err, body) => {
+                    assert.ifError(err);
+                    const metricTypes = ['backlog', 'completions',
+                        'throughput', 'failures'];
+                    const keys = Object.keys(body);
+                    keys.forEach(key => {
```

Comment:
Seems like maybe we would want to iterate through `metricTypes` and comparing them to the object body so that we can catch a case where a metric is missing from the response.